test/named-routes-test.js

```diff
@@ -167,22 +167,22 @@ describe('NamedURLResolver', function() {
             React.createElement(Route, {name: 'test9', path: '/path(/:param1/:param2/)'}),
         ]);
 
-        expect(resolver.resolve("test1", {param: 1})).to.equal("/path/1", "Simple omit case 1");
-        expect(resolver.resolve("test2", {param: 1})).to.equal("/path/1", "Simple omit case 2");
+        expect(resolver.resolve("test1")).to.equal("/path", "Simple omit case 1");
+        expect(resolver.resolve("test2")).to.equal("/path", "Simple omit case 2");
```

Comment:
It would be best to keep those two old test cases and just add new ones next to them